### ISSUE TITLE
Update react versions

### ISSUE THREAD

**Issue by santialbo** [TeamWertarbyte/material-ui-password-field#14]
Could you update the versions of your material-ui components to something like
```
  "peerDependencies": {
    "react": "^15.3.0 || ^16.0.0",
    "react-dom": "^15.3.0 || ^16.0.0"
  },
```
Thanks for the great work and for opensourcing it.

**Comment by leMaik** [comment]
@santialbo  Does this component actually work with react 16?

**Comment by santialbo** [comment]
I'm using the following without any issues:
```
    "material-ui-image": "^1.4.1",
    "material-ui-password-field": "^1.3.0",
    "material-ui-search-bar": "^0.4.0",
    "material-ui-toggle-icon": "^0.2.0"
```

The [breaking changes in v16](https://reactjs.org/blog/2017/09/26/react-v16.0.html#breaking-changes) are very uncommon usages of their apis. I would expect simple components like these to work straighaway with v16.

**Comment by leMaik** [comment]
@santialbo Wow, you're using more of our components than we do in most of our apps. :smile: I'll update the peer dependency soon, but I can't get the storybook to work with react 16 at the moment. :roll_eyes:

**Comment by leMaik** [comment]
Fixed in 53b2f4d, I'll publish a new release soon.
